openqabot/pc_helper.py

```diff
@@ -107,9 +75,13 @@ def apply_publiccloud_pint_image(settings):
         settings["PUBLIC_CLOUD_IMAGE_ID"] = image[settings["PUBLIC_CLOUD_PINT_FIELD"]]
         settings["PUBLIC_CLOUD_IMAGE_NAME"] = image["name"]
         settings["PUBLIC_CLOUD_IMAGE_STATE"] = image["state"]
-        # Remove pint query settings. They are not required in the scheduled job
-        if "PUBLIC_CLOUD_PINT_QUERY" in settings:
-            del settings["PUBLIC_CLOUD_PINT_QUERY"]
+    except BaseException as e:
+        log_error = 'PUBLIC_CLOUD_PINT_QUERY handling failed'
+        if 'PUBLIC_CLOUD_PINT_NAME' in settings:
+            log_error += f'for {settings["PUBLIC_CLOUD_PINT_NAME"]}'
```

Comment:
```suggestion
            log_error += f' for {settings["PUBLIC_CLOUD_PINT_NAME"]}'
```

Add a space to get a proper error message.